### PR TITLE
add stats_can

### PR DESCRIPTION
**Pull request by ianepreston**
Checklist
- [X ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ X] Source is from official source
- [ X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ X] If static libraries are linked in, the license of the static library is packaged.
- [X ] Build number is 0
- [ X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ianepreston** [comment]
@conda-forge-admin, please ping team

**Comment by ianepreston** [comment]
I'm not sure why this is failing. I took a look at the build log and it can't find poetry. I did build the package using poetry, but I'm not sure why it would require it to install it, or what I would have to change in order to get it to build

File: recipes/stats_can/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python 3
```
Comment (scopatz) [inline recipes/stats_can/meta.yaml:22]: This should be `python >=3.0` or similar and the same line should be used in host

**Comment by ianepreston** [comment]
I'm not sure where to go forward on troubleshooting this. I feel like just pushing up little tweaks to ```meta.yaml``` and waiting for the recipe to fail is a bad use of resources. The particular failure seems to be around poetry: [ModuleNotFoundError: No module named 'poetry'](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=202885&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=610)

I took a look at the sdist that this is building from and it has a ```setup.py``` file that shouldn't need poetry. Even if it's trying to use the ```pyproject.toml``` I believe recent versions of pip should be able to handle installing packages without having poetry installed. I'm at a loss for next steps here. @scopatz, I incorporated your suggestion around pinning python versions, but I think that was just an unrelated suggestion to improve the recipe. Any suggestions on where I can go from here? Googling "poetry" and "conda-forge" just turns up a lot of links for installing poetry with conda or using conda environments in poetry.

Thanks very much

File: recipes/stats_can/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  host:
+    - python >=3.0
+    - pip
```
Comment (scopatz) [inline recipes/stats_can/meta.yaml:20]: You might need `poetry` as a dep here?

Comment (ianepreston) [reply recipes/stats_can/meta.yaml:20]: I gave that a shot in build, but I'll try again in host. I have to admit I don't have a good mental model about the distinction between those two categories.

Comment (ianepreston) [reply recipes/stats_can/meta.yaml:20]: Yup, putting it in host fixed it >.<
Guess I should have fiddled with one more thing before tagging in for help.
I appreciate the reply!

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**